charts/spire/charts/spire-server/values.yaml

```
@@ -167,7 +167,9 @@ dataStore:
     username: spire
     ## @param dataStore.sql.password Only used when type != "sqlite3"
     password: ""
-    ## @param dataStore.sql.options [array] Only used when type != "sqlite3"
+    ## @param dataStore.sql.file Only used when type == "sqlite3"
```

```suggestion
    ## @param dataStore.sql.file Data source file. Only used when type == "sqlite3"
```

add extra info about the file parameter (obviously need to regen the readme if accepted)